deps.ffmpeg/60-lame.ps1

```diff
@@ -19,10 +19,11 @@ function Setup {
 
 function Clean {
     Set-Location "${Name}-${Version}"
-    if ( Test-Path "build_${Target}" ) {
-        Log-Information "Clean build directory (${Target})"
-        Remove-Item -Path "build_${Target}" -Recurse -Force
-    }
+
+    Get-ChildItem -Recurse -Include 'lame.exe','mp3x.exe','mp3rtp.exe' | Remove-Item
+    Get-ChildItem -Recurse -Include 'libmp3lame.*','libmp3lame-static.lib','lame_enc.dll' | Remove-Item
+    Get-ChildItem -Recurse -Include 'lame.pdb','icl.pch' | Remove-Item
+    Get-ChildItem -Recurse -Include '*.obj','*.res' | Remove-Item
```

Comment:
Noting for posterity that this is the correct syntax due to a bug in `Remove-Item`:
* https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/remove-item?view=powershell-7.4#example-4-delete-files-in-subfolders-recursively